CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 - Add `main` flag to threads and `in_foreground` flag for app contexts  ([#2516](https://github.com/getsentry/sentry-java/pull/2516))
+- Add `enableTracing` option ([#2530](https://github.com/getsentry/sentry-java/pull/2530))
```

Comment:
`m`: can we elaborate a little bit more for the changelog readers how it behaves? (i.e. it does not change existing default behavior, etc.)

Reply:
Will do

Reply:
updated